manifest/docker_schema2.go

```diff
@@ -151,11 +152,11 @@ type Schema2History struct {
 // Schema2Image is an Image in docker/docker/image.
 type Schema2Image struct {
 	Schema2V1Image
-	Parent     digest.Digest    `json:"parent,omitempty"`
-	RootFS     *Schema2RootFS   `json:"rootfs,omitempty"`
-	History    []Schema2History `json:"history,omitempty"`
-	OSVersion  string           `json:"os.version,omitempty"`
-	OSFeatures []string         `json:"os.features,omitempty"`
+	Parent     digest.Digest       `json:"parent,omitempty"`
+	RootFS     *Schema2RootFS      `json:"rootfs,omitempty"`
+	History    []imgspecv1.History `json:"history,omitempty"`
```

Comment:
This is an externally-visible breaking change to the API, so not something we can do. (Also, inaccurate, if either of the formats added more fields.)